tests/manual-test-cases/Group19-ROBO/19-5-ROBO-Vcenter-Connectivity.md

```diff
@@ -0,0 +1,35 @@
+Test 19-5 - ROBO vCenter Connectivity
+=======
+
+# Purpose:
+To verify that the applications deployed in containerVMs in a ROBO Advanced environment are functional when the ESXi(s) hosting the containerVMs are disconnected from the vSphere host. This test exercises the WAN connectivity and resiliency support for a ROBO environment that could represent a customer's cluster topology. 
+
+# References:
+1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
+* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
+* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+
+See https://confluence.eng.vmware.com/display/CNA/VIC+ROBO for more details.
+
+# Test Steps:
+1. Deploy a ROBO Advanced vCenter testbed for both environments above
+2. Install the VIC appliance on vCenter
+3. Create and start some container services such as nginx, wordpress or a database
+4. Run a containerized application with docker-compose
+5. For each ESXi host that hosts containerVM(s), disconnect it from vCenter
```

Comment:
If we're emulating WAN link outage then it should be all hosts in the cluster.
We should also ensure that the hosts can continue to talk to each other.

I'm unsure what is meant by `disconnect` - it needs to be _unexpected_ from both the VC and ESX side so that we don't end up with polite behaviours.

Might be possible with firewall rules on ESX or in nimbus. Alternatively if the VC is addressed via a separate network than the other hosts in the cluster.